### PR TITLE
Remove extraneous append of exception message.

### PR DESCRIPTION
**Pull request by danielmoy-google**

### CODE REVIEW

File: kythe/go/platform/tools/extraction/repotester.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ var (
 	repos      = flag.String("repos", "", "A comma delimited list of repos to test")
 	reposFile  = flag.String("repo_list_file", "", "A file that contains a newline delimited list of repos to test")
 	configPath = flag.String("config", "", "An optional config file to specify kythe.proto.ExtractionConfiguration logic")
+	index      = flag.Bool("index", false, "Whether to attempt to index a repo after successful extraction")
```
Comment (jrtom) [inline kythe/go/platform/tools/extraction/repotester.go]: Is this related to this PR?

Comment (danielmoy-google) [reply kythe/go/platform/tools/extraction/repotester.go]: ah damnit I pulled in the wrong stuff.  Thanks for catching that.

**Comment by danielmoy-google** [comment]
If anyone objects to the back and forth I can just kill this and re-do it with only the correct file.

**Comment by danielmoy-google** [comment]
That was my theory
